app/lib/scraper.rb

```diff
@@ -0,0 +1,62 @@
+class Scraper
+
+
+
+  def self.fetch
+
+
+    journal_feeds = JournalFeed.all
+
+    feed = Feedjira::Feed.fetch_and_parse(journal_feeds.last.url)
+    binding.pry
+    journal_feeds.each do |journal_feed|
+      feed = Feedjira::Feed.fetch_and_parse(journal_feed.url)
+      journal = Journal.new({journal_feed: journal_feed, title: journal_feed.title, date: feed.entries.first.published })
+      unless Scraper.dates(journal_feed).include?(journal.date)
+        feed.entries.each do |entry|
+          unless entry.summary.blank?
```

Comment:
this would be easier to read/understand if it ware written with postiive logic as `if entry.summary.present?`